src/tiled/commanddatamodel.h

```diff
@@ -143,7 +129,15 @@ class CommandDataModel : public QAbstractTableModel
 
     QKeySequence shortcut(const QModelIndex &index) const;
 
-    void setShortcut(const QModelIndex &index, const QKeySequence &keySequence);
+    void setShortcut(const QModelIndex &index, const QKeySequence &value);
+
+    bool saveBeforeExecute(const QModelIndex &index) const;
+
+    void setSaveBeforeExecute(const QModelIndex &index, const bool &value);
+
+    QString command(const QModelIndex &index) const;
```

Comment:
What about making this function return the full Command, like `firstEnabledCommand` is doing, avoiding the need for all the other getters?